kernel/KernelMain.cpp

```diff
@@ -204,10 +205,11 @@ namespace Kernel
 
         InitPanic();
         IoApic::InitAll();
+        uint64_t unixTime = Devices::ReadRtcTime();
```

Comment:
Nice! Let's put this to use, instead of printing the time we should store it in the system clock. You can use
`SetBootEpoch(uint64_t)` to set the current time of the system clock. SystemClock.h is already included.